lib/main_common.pm

```diff
@@ -1172,6 +1172,7 @@ sub load_consoletests {
         loadtest "console/zypper_ref";
     }
     loadtest "console/ncurses";
+    loadtest "console/ncurses_extended";
```

Comment:
Done

Reply:
Sorry, I don't see that as the line still shows up here in the generic console tests. Also, please reference a TW verification run as well.

Reply:
Yes, I missed 12 SP3. Solved

Reply:
I don't know what your comment has anything to do with this line of code.

Reply:
Yes, you are right It was a mistake, I was confused, I fixed he schedule configs, but not from here. But is it necessary to be here having the schedule configs?

Reply:
Well, lib/main_common.pm is not evaluated if you define a YAML schedule document. But some products, e.g. openSUSE Tumbleweed, use lib/main_common.pm . About the current status of SLE tests I am not fully aware. Neverthless having https://opensource.suse.com/ in mind we should always aim to have Tumbleweed tests which can actually save us a lot of work because additions to Factory which is the base for all other derived products are then tested and regression caught as early as possible.